pynestml/codegeneration/nest_codegenerator.py

```diff
@@ -807,6 +807,12 @@ def transform_ode_and_kernels_to_json(self, neuron: ASTNeuron, parameters_block,
                     odetoolbox_indict["parameters"][var.get_complete_name(
                     )] = gsl_printer.print_expression(decl.get_expression())
 
+        # add "current" type input ports to parameters with zero value, to prevent unknown terms from preventing an analytic solution
```

Comment:
I don't understand the comment. What to you mean by "adding input ports to parameters"?